landoapi/stacks.py

```diff
@@ -42,35 +38,21 @@ def build_stack_graph(
         between two nodes. `child` and `parent` are also string
         PHIDs.
     """
-    phids = set()
-    new_phids = {revision_phid}
+    result = phab.call_conduit(
+        "differential.revision.search",
+        constraints={"phids": [revision_phid]},
+        limit=1,
+    )
+
+    stack_graph = result["data"][0]["fields"]["stackGraph"]
```

Comment:
This should use `phab.single` and `phab.expect`.

Reply:
Done.